pipeline/unified/channel/sign-release-package-windows.yaml

```diff
@@ -11,6 +11,7 @@ jobs:
           unsignedPipelineResource: ${{ parameters.unsignedPipelineResource }}
           unsignedArtifactName: ${{ parameters.unsignedArtifactName }}
           signedArtifactName: ${{ parameters.signedArtifactName }}
+          channel: ${{ parameters.channel }}
```

Comment:
Should this be added as a parameter of this template?

Reply:
Good catch, yes!